scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -43,32 +41,30 @@ workflow GvsImportGenomes {
       preemptible_tries = preemptible_tries
   }
 
-  call GetMaxTableId {
-    input:
-      sample_map = sample_map
+  if (defined(sample_map)) {
```

Comment:
What do we need the legacy mode for?

Reply:
will leaving this in be helpful for anyone? I wasn't sure if people had tests that used a sample map file. Although, this won't update the sample_info table anymore, so maybe I should just remove it. If anyone else has an opinion, let me know.